### ISSUE TITLE
False positive on handle-callback-err when not using parentheses

### ISSUE THREAD

**Issue by fxlemire** [Microsoft/vscode-tslint#150]
```
tslint: 4.0.2
typescript: 2.1.4
vscode: 1.7.2
vscode-tslint: 0.7.1
```

Following code displays a `handle-callback-err` inside VS Code, but does not if `err` is wrapped with parentheses.
Error not displayed when executing `tslint` from the command line.

```
app.listen(PORT, err => {
  if (err) {
    console.log(err);
    return;
  }
}
```

rule: `"handle-callback-err": [true, "^(err|error)$"]`

**Comment by egamma** [comment]
@fxlemire Can you pls confirm that the error is from tslint and not not eslint? You can find this out by inspecting the prefix of the warning, e.g., [tslint] or [eslint].

According https://github.com/palantir/tslint/issues/1379 it appears that this rule is not yet supported by tslint, but eslint supports this rule.

**Comment by fxlemire** [comment]
@egamma the prefix is indeed [tslint], which is what confused me. I just realized that this rule actually comes from [tslint-eslint-rules](https://github.com/buzinas/tslint-eslint-rules)... my bad.

Closing the issue, sorry about that.
